client-app/shared/cart/components/cart-line-items.vue

```diff
@@ -71,6 +71,7 @@ interface IProps {
   readonly?: boolean;
   items?: LineItemType[];
   validationErrors?: ValidationErrorType[];
+  selectable?: boolean;
```

Comment:
set default true
then in your code selectable="false"

Reply:
That's keep backward compatibility for sure, but will against our own and vue's conventions where boolean props are always false by default to allow their use with other components as just
```html
<input property>
```
instead of
```html
<input property="value">
```

Reply:
And you already accepted https://github.com/VirtoCommerce/vc-frontend/pull/1446 with the same change

Reply:
@asvishnyakov 
Couldn't find the explicit mention that default value shouldn't be `true`. I mean yes, it makes sense, especially when creating new component. But don't see any problems in doing that when modifying existing component to avoid breaking changes. How could it harm?

Reply:
@ivan-kalachikov It couldn't, it cost me nothing, I just can't understand then why you accepted #1446 as is, while asked to change this one.